megamek/src/megamek/client/ratgenerator/ModelRecord.java

```diff
@@ -246,23 +172,53 @@ public int getNetworkMask() {
     public void setNetwork(int network) {
         this.networkMask = network;
     }
+
+    /**
+     * Proportion of total weapons BV that is optimized against airborne targets
+     * @return   between zero (none) and 1.0 (all weapons)
+     */
     public double getFlak() {
-        return flak;
+        return flakBVProportion;
     }
-    public void setFlak(double flak) {
-        this.flak = flak;
+
+    /**
+     * Proportion of total weapons BV that is artillery
+     * @return   between zero (none) and 1.0 (all weapons)
+     */
+    public double getArtilleryProportion () {
+        return artilleryBVProportion;
     }
 
+    /**
+     * Proportion of total weapons BV that is best at long range
+     * @return   between zero (none) and 1.0 (all weapons)
+     */
     public double getLongRange() {
-        return longRange;
+        return lrBVProportion;
+    }
+
+    /**
+     * Proportion of total weapons BV that is best at short range
+     * @return   between zero (none) and 1.o (all weapons)
+     */
+    public double getSRProportion() {
+        return srBVProportion;
     }
 
     public int getSpeed() {
         return speed;
     }
 
+    public boolean getJump() {
+        return canJump;
```

Comment:
I didnt check if canJump is used elsewhere.

Reply:
This follows a similar style to the other getters, referring to a backing field rather than calling the MechSummary object, for consistency. Easy enough to change, though.

Reply:
It's alright as it is. Just something I thought while reading.